src/components/sendV2/summary/summary.js

```diff
@@ -24,22 +26,76 @@ class Summary extends React.Component {
       },
     };
 
-    this.prevStep = this.prevStep.bind(this);
-    this.nextStep = this.nextStep.bind(this);
+    this.checkForHardwareWallet = this.checkForHardwareWallet.bind(this);
     this.checkSecondPassphrase = this.checkSecondPassphrase.bind(this);
-    this.amountAndFee = this.amountAndFee.bind(this);
+    this.nextStep = this.nextStep.bind(this);
+    this.prevStep = this.prevStep.bind(this);
+    this.submitTransaction = this.submitTransaction.bind(this);
   }
 
   componentDidMount() {
     const { account } = this.props;
+    let newState = {};
 
     // istanbul ignore else
     if (account && account.secondPublicKey) {
-      this.setState({
+      newState = {
+        ...this.state,
         secondPassphrase: {
           ...this.state.secondPassphrase,
           hasSecondPassphrase: true,
         },
+      };
+    }
+
+    // istanbul ignore else
+    if (account.hwInfo && account.hwInfo.deviceId) {
+      newState = {
+        ...this.state,
+        isHardwareWalletConnected: true,
+        isLoading: true,
+      };
+      this.submitTransaction();
+    }
+
+    this.setState({ ...newState });
+  }
+
+  componentDidUpdate() {
+    this.checkForHardwareWallet();
+  }
+
+  submitTransaction() {
+    this.props.sent({
+      account: this.props.account,
+      recipientId: this.props.fields.recipient.address,
+      amount: this.props.fields.amount.value,
+      data: this.props.fields.reference.value,
+      passphrase: this.props.account.passphrase,
+      secondPassphrase: this.state.secondPassphrase.value,
+    });
+  }
+
+  checkForHardwareWallet() {
+    const { failedTransactions, pendingTransactions } = this.props;
+    const { isHardwareWalletConnected, isLoading } = this.state;
+
+    const isATransactionPending = isHardwareWalletConnected
```

Comment:
maybe rename for `hasPendingTransaction`, a better name IMO

Reply:
you are right, I changed it